wnprc_billing/resources/queries/ehrLinked/investigators.query.xml

```diff
@@ -0,0 +1,20 @@
+<query xmlns="http://labkey.org/data/xml/query">
+    <metadata>
+        <tables xmlns="http://labkey.org/data/xml">
+            <table tableName="investigators" tableDbType="TABLE">
+                <tableTitle>Investigators</tableTitle>
+                <titleColumn>lastName</titleColumn>
```

Comment:
This could point to the proposed magic calculated column

Reply:
Remove titleColumn since no longer will be using lastName.